Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -262,7 +262,10 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
 
     override open func viewDidDisappear(_ animated: Bool) {
         super.viewDidDisappear(animated)
-
+        
+        // We need to cleanup the state when composer not to type when it is not presented.
+        channelController.sendStopTypingEvent()
```

Comment:
Implementing `stopTyping` inside `EventsSender`'s `deinit` feels like unnecessary call.. But this is up to discussion since `EventsSender` is used only for sending `typingEvents`... Maybe rename it?

Reply:
Resolved by renaming `EventsSender` to `TypingEventsSender` and implementing on `deinit`